diff -u doxygen-1.8.10/CMakeLists.txt doxygen-1.8.10.new/CMakeLists.txt
--- doxygen-1.8.10/CMakeLists.txt     2015-06-16 12:42:32.000000000 -0500
+++ doxygen-1.8.10.new/CMakeLists.txt 2016-03-14 17:35:26.000000000 -0500
@@ -39,7 +39,6 @@
 endif()
 
 if (${CMAKE_SYSTEM} MATCHES "Darwin")
-    set(CMAKE_CXX_FLAGS "-Wno-deprecated-register -mmacosx-version-min=10.5 ${CMAKE_CXX_FLAGS}")
     find_library(CORESERVICES_LIB CoreServices)
     set(EXTRA_LIBS ${CORESERVICES_LIB})
 endif()
@@ -61,9 +60,8 @@
     find_package(SQLite3 REQUIRED)
 endif()
 
-find_package(Iconv REQUIRED)
-include_directories(${ICONV_INCLUDE_DIR})
-
+set(ICONV_LIBRARIES "/usr/lib/libiconv.dylib" CACHE FILEPATH "Iconv Library")
+mark_as_advanced(ICONV_LIBRARIES)
 
 #set(DOXYDOCS ${CMAKE_SOURCE_DIR}/doc CACHE INTERNAL "Path to doxygen docs")
 set(DOC_INSTALL_DIR "share/doc/packages/doxygen" CACHE STRING "Relative path where to install the documentation")
